.circleci/config.yml

```diff
@@ -658,6 +712,13 @@ workflows:
           context: Slack Orb
       - spm-core-integration-test-job:
           name: "Xcode 13; iOS 15.0; SPM Core test"
+      - generate-docs-job:
+          name: Generate docs and push it for release tag
+          context: Slack Orb
+          filters:
+            tags:
+              only: /.*/
```

Comment:
Do you want to filter out this build in regular branches? As it these lines:
```yaml
branches:
  ignore: /.*/
```

Reply:
This was a way to trigger the job for all the tags. Reading docs closely, I understood that we can put semver filter here, it will still trigger job for all the branch pushes.